src/server/server.js

```diff
@@ -94,11 +94,12 @@ io.sockets.on("connection", (socket) => {
   });
 
   // Propose Vote
-  socket.on("propose-vote", ({ voteId }) => {
+  socket.on("propose-vote", ({ voteId, link }) => {
     // console.log('[server] propose-vote', socket.id);
     rooms[currentRoomId].currentVote = {
       id: voteId,
       userId: socket.id,
+      link,
```

Comment:
```suggestion
      link: !!process.env.JIRA_PATH && `${process.env.JIRA_PATH}vodeId`,
```

Reply:
i rather have a process env than a extra field (keep it easy for the users)

Reply:
add --env JIRA_PATH=https://jira.server.com/browse/ in docker run command on README.md

Reply:
I began to do it like this but I was not sure this project was aimed to be used only on JIRA or with a issue tracker that had this kind of link structure.

Seems ok to me, I'll do the change